src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -69,11 +69,6 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </summary>
         public string LeaseDatabaseName { get; set; }
 
-        /// <summary>
-        /// Gets or sets the lease options for this particular DocumentDB Trigger. Overrides any value defined in <see cref="DocumentDBConfiguration" /> 
-        /// </summary>
-        public ChangeFeedHostOptions LeaseOptions { get; set; }
```

Comment:
This trigger isn't considered GA yet, is it? If so, pulling this property away would be a breaking change.

Reply:
This property was impossible to set, it should not have gone out in the first place. You could not create a function that passed that attribute because it didn't compile:
`CosmosDBTrigger("mydb", "mycollection", ChangeFeedHostOptions=new ChangeFeedHostOptions())`

Said that ChangeFeedHostOptions was not a valid parameter type. Apparently, only primitive types can be defined (string, int, etc), its a CLR restriction.